src/modules/taskHelper.js

```diff
@@ -0,0 +1,53 @@
+class TaskHelpers {
+  static CreateBookEntry(existingTasks, taskDescription) {
+    const id = existingTasks.length === 0 ? 0 : existingTasks.length;
+    const updatedTasks = existingTasks;
+
+    updatedTasks.push({
+      index: id,
+      description: taskDescription,
+      completed: false,
+      cleared: false,
+    });
+
+    return updatedTasks;
+  }
+
+  static UpdateTaskStore = (tasks) => {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  };
+
+  static RetrieveTasks = () => {
+    try {
+      Array.from((JSON.parse(localStorage.getItem('tasks'))));
+    } catch (err) {
+      localStorage.setItem('tasks', JSON.stringify([]));
+    }
+    return JSON.parse(localStorage.getItem('tasks'));
+  };
```

Comment:
- [ ] After I marked a task as completed and refreshed my browser the task completion key went back to false. It would be great if a user could retrieve the same list of tasks with their correct completion key on reload. Kindly fix this too.